tests/test_find_iocs.py

```diff
@@ -15,13 +15,13 @@ def test_find_iocs(text: str, results: Dict, args: Dict) -> None:
 
     # Get all keys
     ioc_finder_keys = _get_nonempty_keys(iocs)
-    results_keys = list(results.keys())
+    results_keys = _get_nonempty_keys(results)
 
     # Test output with expected results
     assert sorted(ioc_finder_keys) == sorted(
         results_keys
     ), f"Key values don't match '{ioc_finder_keys}' vs '{results_keys}'"
-    for key in ioc_finder_keys:
+    for key in results.keys():
```

Comment:
What is the purpose of this change?

From what I can tell, the code would work the same way if it were left as it was:

```suggestion
    for key in ioc_finder_keys:
```